force_bdss/tests/dummy_classes/interface.py

```diff
@@ -0,0 +1,5 @@
+from traits.api import Interface
+
+
+class DummyInterface(Interface):
```

Comment:
This class has been included in a separate file to `DummyServiceOffersPlugin` in order to mirror the typical procedure for loading UI modules.